src/BlazorWebView/tests/MauiDeviceTests/MauiBlazorWebView.DeviceTests.csproj

```diff
@@ -7,6 +7,8 @@
     <RootNamespace>Microsoft.Maui.MauiBlazorWebView.DeviceTests</RootNamespace>
     <AssemblyName>Microsoft.Maui.MauiBlazorWebView.DeviceTests</AssemblyName>
     <NoWarn>$(NoWarn),CA1416</NoWarn>
+    <!-- Disable multi-RID builds to workaround a parallel build issue -->
```

Comment:
Is there an issue tracking this so we can remove when it is fixed?

Reply:
Rui had one, but I think it was conflated with an Android issue (it's now closed).

I filed a new one: https://github.com/xamarin/xamarin-macios/issues/17497